gt/main.c

```diff
@@ -1894,21 +1894,34 @@ init_gt_instances(struct gt_config *gt_conf)
 	int ret;
 	int num_succ_instances = 0;
 	struct gt_instance *inst_ptr;
+	/* Take the GGU packets into account. */
+	unsigned int num_mbuf = calculate_mempool_config_para("gt",
+		gt_conf->net, gt_conf->net->front.total_pkt_burst +
+		gt_conf->net->back.total_pkt_burst +
+		gt_conf->frag_max_entries);
```

Comment:
What's the relationship between `gt_conf->frag_max_entries` and GGU packets?

Reply:
As the GT blocks call ``process_death_row()`` to process the expired packets. In the worst case, ``process_death_row()`` needs to notify Gatekeeper the decisions about all the packets in the fragmentation table via GGU packets. One thing to note that one GGU packet can carry multiple policy decisions.

Reply:
Your justification is not simple, so it should be written as a comment to explain it.

Also, what about the GGU packets that GT normally sends out? Shouldn't we account for them as well?

Reply:
Will do.

The GGU packets that GT normally sends out should be ``gt_conf->max_pkt_burst``, we should take them into account as well.

Moreover, the ``gt_conf->net->back.total_pkt_burst`` is not needed here as GT blocks only run on the front NIC.

Reply:
Neither a comment summarizing the explanation above, nor `gt_conf->max_pkt_burst` has been added to the code.

Reply:
Ops, my fault. I was reading the wrong spot in the code. Disregard my last comment.